pkg/virt-api/rest/authorizer_test.go

```diff
@@ -26,21 +26,21 @@ import (
 	"net/http"
 	"net/url"
 
-	restful "github.com/emicklei/go-restful"
+	"github.com/emicklei/go-restful"
```

Comment:
Similarly, can you move the refactoring to a separate commit?

Reply:
Done